md/terminusx/start-with-a-client.md

```diff
@@ -465,7 +465,7 @@ print(list(matches))
 
 #### Code: Query documents using WOQL
 
-Query documents using Web Object Query Language(WOQL) to get same result given by the above example. You can find more about WOQL [here](https://terminusdb.com/docs/main/#/explanation/explanation-woql).
+Query documents using Web Object Query Language (WOQL) to get same result given by the above example. You can find more about WOQL [here](/#/explanation/explanation-woql).

```

Comment:
The link `/#/explanation/explanation-woql` doesn't work. The way that I found the correct link format is by looking through the repository for other `explanation` links. Try this in your local `terminusdb-docs` repository:

```sh
git grep explanation
```

Look in the results for the Markdown link pattern, `[...](...)`, and you'll find many examples.

Reply:
Actually i was using online editor of github to make the PR so i didn't check it locally 😅

Reply:
Done. Thank you for pointing it out